src/bundle/Resources/public/scss/_inputs.scss

```diff
@@ -0,0 +1,67 @@
+.form-control {
+    &--is-invalid {
+        @include invalid-text-input;
+    }
+}
+
+.ez-input {
+    outline: none;
+    transition: all 0.2s $ez-admin-transition;
+
+    &--checkbox {
+        -webkit-appearance: none;
+        -moz-appearance: none;
+        appearance: none;
+        position: relative;
+        display: inline-block;
+        width: calculateRem(16px);
+        height: calculateRem(16px);
+        background-color: $ez-white;
+        border-radius: $ibexa-border-radius;
+        border: calculateRem(1px) solid $ibexa-color-base-dark;
+
+        &:active {
+            @include input-active;
+        }
+
+        &:checked {
+            border: calculateRem(1px) solid  $ibexa-color-primary;
+            background-color:  $ibexa-color-primary;
+        }
+
+        &:checked:after {
```

Comment:
This could be moved above, into `&:checked` and be just `&:after` but it's little nitpicking